admin-frontend/src/menu.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { MenuItemLink } from 'react-admin';
+import styled from 'styled-components';
+import ChildCareIcon from '@material-ui/icons/ChildCare';
+import ListIcon from '@material-ui/icons/ViewList';
+import NewSeasonIcon from '@material-ui/icons/Autorenew';
+import usePermissions from './hooks/usePermissions';
+
+const MenuContainer = styled.div`
+  margin-top: 1.5em;
+`;
+
+const Menu = () => {
+  const { isSuperAdmin } = usePermissions();
+
+  return (
+    <MenuContainer>
+      <MenuItemLink
+        to="/junior"
+        primaryText="Nuoret"
+        leftIcon={<ChildCareIcon />}
+      />
+      <MenuItemLink
+        to="/youthClub"
+        primaryText="Nuorisotilat"
+        leftIcon={<ListIcon />}
+      />
+      {isSuperAdmin && (
```

Comment:
From last meeting discussion, admin aka youth worker, should also be able to edit junior as well. However here that feature only available to super admin, is this intended?